src/main/java/ladder/Main.java

```diff
@@ -0,0 +1,21 @@
+package ladder;
+
+import ladder.domain.Ladder;
+import ladder.domain.People;
+import ladder.view.InputView;
+import ladder.view.ResultView;
+
+import java.util.Random;
+
+public class Main {
+    private static void makeLadder() {
+        People people = InputView.inputPeople();
+        int ladderHeight = InputView.inputLadderHeight();
+        Ladder ladder = new Ladder(ladderHeight, people, (idx, preConnected) -> !preConnected && new Random().nextBoolean());
```

Comment:
일부러 람다 사용해보려고 ConnectStrategy를 람다로 표현했습니다. 그런데 오히려 코드 읽기가 어려워지는 거 같아서.. 이런 경우엔 따로 클래스로 빼는 게 나을까요?

Reply:
클래스도 좋고 변수로 선언해도 좋을 것 같아요
```suggestion
        final ConnectStrategy defaultStrategy = (idx, preConnected) -> !preConnected && RANDOM.nextBoolean();
        final Ladder ladder = new Ladder(ladderHeight, people, defaultStrategy);
```

---
>Optional을 어디에 써야할지 감이 잘 오지 않습니다. 적용하기 좋은 곳이 있을까요?

null을 활용한 설계가 아니라면 억지로 사용하지 않으셔도 됩니다 😃 
기능 목록을 정의하실 때 `마지막 포인트는 null 이다.` 라는 내용이 있었는데요.
실제로 null을 활용했다면 Optional을 활용할 수 있는 지점이 아닐까 싶어요.